app/models/ability.rb

```diff
@@ -0,0 +1,12 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    if user.is? :admin
+      can :manage, :all
+    else
+      can :destroy, Post, author_id: user.id
+      can :destroy, Comment, author_id: user.id
```

Comment:
- [ ] After these lines it would be good if you could also add ` can :read, :all` meaning that every user can read the posts